rdt-open-reader/src/test/java/io/ona/rdt/robolectric/util/UtilsTest.java

```diff
@@ -223,4 +224,11 @@ public void testIsValidJSONObjectShouldReturnCorrectStatus() {
         Assert.assertFalse(Utils.isValidJSONObject(null));
         Assert.assertFalse(Utils.isValidJSONObject(""));
     }
+
+    @Test
+    public void testVerifyUserAuthorizationShouldExecuteAuthorizationTask() {
+        Assert.assertEquals(AsyncTask.Status.PENDING, Utils.UserAuthorizationVerificationTask.getInstance(RDTApplication.getInstance()).getStatus());
+        Utils.verifyUserAuthorization(RDTApplication.getInstance());
+        Assert.assertEquals(AsyncTask.Status.FINISHED, Utils.UserAuthorizationVerificationTask.getInstance(RDTApplication.getInstance()).getStatus());
```

Comment:
Revert the changes here to what was there before. We only need to remove the line referenced in the comment https://github.com/onaio/rdt-standard/pull/945#discussion_r607620352.

And make sure to call https://github.com/onaio/rdt-standard/pull/945#discussion_r607619746 as in the comment.

Reply:
changes updated.